### PR TITLE
LTD-2286: Changed phone -> telephone for consistency

### PR DESCRIPTION
**Pull request by antroy-madetech**

### CODE REVIEW

File: exporter/core/organisation/validators.py [hunk 1]
```diff
@@ -6,6 +7,8 @@
 
 from .constants import Validation
 
+logger = logging.getLogger(__name__)
```
Comment (Harriethw) [inline exporter/core/organisation/validators.py:10]: was this logging import and logger left in accidentally?

Comment (antroy-madetech) [reply exporter/core/organisation/validators.py:10]: yep.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,3 +1,6 @@
+default_language_version:
+    python: python3.7
+
```
[the remark this answers is not in the document]
Comment (antroy-madetech) [reply .pre-commit-config.yaml:3]: Not yet - Kevin's tested on Mac. Was going to ask for a Windows volunteer before merging.

[the remark this answers is not in the document]
Comment (hnryjmes) [reply .pre-commit-config.yaml:3]: I think I tried this at some point last month, it then started a whole team conversation about how exactly we should do pre-commit on this project which never got resolved properly

[the remark this answers is not in the document]
Comment (hnryjmes) [reply .pre-commit-config.yaml:3]: from what I remember I believe the general feeling was that others have been running pre-commit using a different python to the one they may be using from the project. So e.g. if you have python 3.10 running somewhere on your machine, you can use that, but only for pre-commit (!)

File: unit_tests/exporter/core/organisation/test_forms.py [hunk 3]
```diff
@@ -181,6 +181,19 @@ def test_register_commercial_details_form(data, valid, error):
...
+                "address_line_1": "xyz",
+                "region": "r",
+                "city": "c1",
+                "postcode": "pc",
+                "phone_number": "01234567890",
+                "website": "http://www.notreal.com",
+            },
+            True,
+            None,
+        ),
```
[the remark this answers is not in the document]
Comment (currycoder) [reply unit_tests/exporter/core/organisation/test_forms.py:196]: How about this one @antroy-madetech ?

File: .pre-commit-config.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (antroy-madetech) [reply .pre-commit-config.yaml:3]: Discussion in developers weekly meeting has resulted in this being OK to keep in.

File: unit_tests/exporter/core/organisation/test_forms.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (currycoder) [reply unit_tests/exporter/core/organisation/test_forms.py:196]: My bad, I thought this was a standard django form.  Please disregard.

**Review by currycoder — APPROVED** [review]

**Pull request merged**